.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,6 +6,8 @@ labels: bug, needs triage
 assignees: ''
 
 ---
+If the bug is about the ruleset, consider filing this issue at [axe-windows](https://github.com/microsoft/axe-windows/issues/new/choose) instead.
```

Comment:
I'm not sure "the ruleset" is going to be clear enough for an issue-filer that isn't very familiar with the codebase; maybe expanding on this and/or including an example would be helpful?

Reply:
I'd also rather have a bug in the wrong place than no bug, so maybe worth clarifying that if you aren't sure where to file it, it's okay to use this repo.

Reply:
thanks, will amend!